modules/cuspatial/src/node_cuspatial.ts

```diff
@@ -12,12 +12,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import {Column, FloatingPoint, Int32, Table, Uint32} from '@rapidsai/cudf';
+import {Column, FloatingPoint, Int32, Series, Table, Uint32} from '@rapidsai/cudf';
 import {MemoryResource} from '@rapidsai/rmm';
 
 /** @ignore */
 export declare const _cpp_exports: any;
 
+export type SeriesPair<T extends FloatingPoint> = {
+  x: Series<T>; y: Series<T>;
+}
+
```

Comment:
This type name doesn't seem super descriptive... Can we define it inline? Or rewrite it to use the `Points` type in [geometry.ts](https://github.com/rapidsai/node/blob/2829f0ae92e501eeed192144d049de3a75385ec2/modules/cuspatial/src/geometry.ts#L25)?